pippo-test/pom.xml

```diff
@@ -33,11 +35,23 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.eclipse.jetty.websocket</groupId>
+            <artifactId>websocket-jetty-client</artifactId>
+            <version>${jetty-websocket.version}</version>
+        </dependency>
+
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
             <version>${junit.version}</version>
         </dependency>
+
+        <dependency>
```

Comment:
If a slf4j implementation is needed I prefer to use slf4j-simple that is present as dependency in other pippo module.